src/task/JoinAndReceiveIndexTask.ts

```diff
@@ -75,6 +75,8 @@ export default class JoinAndReceiveIndexTask extends BaseTask {
               context.videoSubscriptionLimit = joinAckFrame.videoSubscriptionLimit;
             }
 
+            context.serverSupportsCompression = joinAckFrame?.wantsCompressedSdp;
```

Comment:
Is `wantsCompressedSdp` correct? Or should it be something like `wantsCompressedSdpAck` or `wantsCompressedSdpConfirmed`?

Reply:
Can we keep it the same as `serverSupportsCompression`?

Reply:
We had decided on wantsCompressedSdp. is there a specific nomenclature for the js SDK?

Reply:
This is correct and already had a lot of media chatter about it :)

Reply:
Ok. If we already had a discussion :))